src/fmt/kv.rs

```diff
@@ -0,0 +1,45 @@
+use std::io::{self, Write};
+
+use super::Formatter;
+use log::kv::{source::Source, Error, Key, Value, Visitor};
+
+/// Format function for serializing key/value pairs
+///
+/// This trait determines how key/value pairs for structured logs are serialized within the default
+/// format.
+pub(crate) type KvFormatFn = dyn Fn(&mut Formatter, &dyn Source) -> io::Result<()> + Sync + Send;
```

Comment:
Is there a reason you didn't match `FormatFn` in making this `Box`ed?

Reply:
Because I use it as a `&` [here](https://github.com/rust-cli/env_logger/pull/137/files#diff-3d324710ba49d50da6915fa5a9bee1c6fbd2a8a97a7d8aa45730c3f9a71f54baR284). But if you would prefer fo rme to make it boxed, I'm fine with that too.

Reply:
Prefer consistency but not strong enough